docs/k8s.md

```diff
@@ -0,0 +1,47 @@
+# kubernetes
+
+With Inago you can easily deploy kubernetes on a coreos cluster.
```

Comment:
on the whole, I'd refer to kubernetes as `Kubernetes`. The same with coreos, we should use `CoreOS`.